app/navigation/routes.js

```diff
@@ -10,52 +25,76 @@ export const RouteTransitions = keyMirror({
 export const Routes = {
     ChannelInfo: {
         key: 'ChannelInfo',
-        title: {id: 'mobile.routes.channelInfo', defaultMessage: 'Info'},
-        transition: RouteTransitions.Horizontal
+        transition: RouteTransitions.Horizontal,
+        component: ChannelInfo,
```

Comment:
Yeah, it's much nicer to have it in one place. Thanks for figuring out the changes that needed to be made to the tests to allow for this, Jarred